layouts/partials/components/carousel.html

```diff
@@ -0,0 +1,42 @@
+<section>
+	<div id="tinyslider-carousel">
+		{{ range sort $.Params.carousel.images "weight" }}
+		<div class="item">
+			<div>
+				<img class="img-responsive" src="{{ .image }}" alt="{{ .description }}" />
+			</div>
+			{{ if .description }}
+			<p>{{ .description }}</p>
+			{{ end }}
+		</div>
+		{{ end }}
+	</div>
+	<div class="tinyslider-controls">
+		<ul class="thumbnails" id="customize-thumbnails">
+		{{ range sort $.Params.carousel.images "weight" }}
+			<li class="item"><img class="img-responsive" src="{{ .image }}" alt="{{ .description }}" width="50px" /></li>
```

Comment:
I can not comment on an image so I comment here.

At present, the newly added images are too large, due to Hugo's restrictions on the size of theme repo. In order to better reuse the existing pictures in the project, you can use the two pictures in the theme assets folder. If you use `utils/get-image` to get image url, you can get these images very easy by setting front matter like `images/hero-left.jpg`.

If you think that two pictures are not enough to show this feature, I can also provide one or two pictures for use.

Reply:
I think we need at least 4 pictures, so that we can show the scrolling and looping. Otherwise we just see swap of two pictures. Feel free to add them to my repo/branch if you have any better. I can severely downgrade quality of the images anyway, as they are only used for demonstration.